stackit/internal/services/resourcemanager/project/resource.go

```diff
@@ -224,12 +225,13 @@ func (r *projectResource) Schema(_ context.Context, _ resource.SchemaRequest, re
 			},
 			"owner_email": schema.StringAttribute{
 				Description: descriptions["owner_email"],
-				// When removing the owner_email field, we should mark the members field as required and add a listvalidator.SizeAtLeast(1) validator to it
-				Optional: true,
+				Required:    true,
```

Comment:
The goal of deprecating the `members` field instead of removing is to not cause a breaking change immediatelly, right?

Reply:
If so we should keep the `owner_email` as optional during the deprecation period, otherwise it will be breaking anyway